updatecli/updatecli.d/jdk11.yaml

```
@@ -32,25 +32,18 @@ sources:
           to: _
 
 conditions:
-  checkTemurinAlpineDockerImage:
-    kind: dockerimage
-    name: Check if the container image "eclipse-temurin:<lastVersion>-jdk-alpine" is available
-    disablesourceinput: true
+  checkTemurinAlpineRelease:
+    name: Check if the "<lastVersion>" is available for alpine
+    kind: shell
     spec:
-      architecture: amd64
-      image: eclipse-temurin
-      tag: '{{source "lastVersion" }}-jdk-alpine'
-  checkTemurinDebianDockerImages:
-    kind: dockerimage
-    name: Check if the container image "eclipse-temurin:<lastVersion>-jdk-jammy" is available
+      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' alpine "x64"
     disablesourceinput: true
+  checkTemurinLinuxRelease:
+    name: Check if the "<lastVersion>" is available for Linux
+    kind: shell
     spec:
-      architectures:
-        - amd64
-        - arm64
-        - s390x
-      image: eclipse-temurin
-      tag: '{{source "lastVersion" }}-jdk-jammy'
+      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' standard "x64,aarch64,s390x"
```

See other comments about comma separator instead of space separator
```suggestion
      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' standard "x64 aarch64 s390x"
```